docs/vgclog_examples.md

```diff
@@ -0,0 +1,594 @@
+
+</gc-op>
+
+<gc-op id="12385" type="classunload" timems="1.607" contextid="12364" timestamp="2020-10-18T13:35:44.609">
+
+<classunload-info classloadercandidates="425" classloadersunloaded="6" classesunloaded="2" anonymousclassesunloaded="1" quiescems="0.000" setupms="1.581" scanms="0.019" postms="0.007" />
+
+</gc-op>
+
+<gc-op id="12386" type="sweep" timems="9.464" contextid="12364" timestamp="2020-10-18T13:35:44.618" />
+```
+
+The end of the increment is recorded with `<gc-end>` and provides another snapshot of memory in the heap, similar to `<gc-start>`. In the example, 60% of the heap now contains free memory as a result of the final global cycle increment. The *nursery* area remains unchanged which means the final increment reclaimed memory only in the *tenure* space, which now has 51% of free memory available. The LOA is now all available as free memory. The SOA has 48% of it's memory available as free memory.
+
+**Note:** The global GC cycle runs to reclaim memory in the *tenure* area. The freeing up of memory in the *nursery* area is achieved by using the partial GC cycle. For more information, see [`gencon` Policy(default)](gc.md#gencon-policy-(default)).
+
+
+```xml
+<gc-end id="12387" type="global" contextid="12364" durationms="24.220" usertimems="86.465" systemtimems="0.000" stalltimems="2.846" timestamp="2020-10-18T13:35:44.618" activeThreads="4">
+
+<mem-info id="12388" free="650476504" total="1073741824" percent="60">
+
+<mem type="nursery" free="235516088" total="268435456" percent="87">
+
+<mem type="allocate" free="235516088" total="241565696" percent="97" />
+
+<mem type="survivor" free="0" total="26869760" percent="0" />
+
+</mem>
+
+<mem type="tenure" free="414960416" total="805306368" percent="51" micro-fragmented="98245682" macro-fragmented="0">
+
+<mem type="soa" free="374694688" total="765040640" percent="48" />
+
+<mem type="loa" free="40265728" total="40265728" percent="100" />
+
+</mem>
+
+<pending-finalizers system="1" default="0" reference="515" classloader="0" />
+
+<remembered-set count="13554" />
+
+</mem-info>
+
+</gc-end>
+```
+
+After the GC operations and sub-operations of the final increment of the global cycle complete, the global cycle ends and the STW pause ends, as shown in the following output:
+
+```xml
+<cycle-end id="12389" type="global" contextid="12364" timestamp="2020-10-18T13:35:44.619" />
+
+<exclusive-end id="12391" timestamp="2020-10-18T13:35:44.619" durationms="24.679" />
+```
+
+### Summary of the example
+
+By analyzing the structure and elements of the example log output, this example shows that the global cycle has the following characteristics:
+
+- The GC global cycle is triggered when a memory threshold is reached and begins with an STW pause
+
+- After the first increment of the GC global cycle completes, the STW pause ends and the second increment runs concurrently.
+
+- While the second increment is running concurrently, a partial GC cycle starts and finishes.
```

Comment:
Actually a few (or none) of scavenges may occur and interleave/overlap. As I read it, it sounds there is exactly 1.
As mentioned in one of discussions, when scavenge runs, concurrent global increment is suspended.